Makefile

```diff
@@ -22,19 +18,18 @@ bs: bootstrap
 ###############################################################################
 # Bootstrapping - get the local machine ready
 
+.PHONY: _bootstrap-install-pnpm
+_bootstrap-install-pnpm:
+	./scripts/dev/install-pnpm.sh
+
 .PHONY: _bootstrap-node
 _bootstrap-node:
-	mkdir -p ${YARN_CACHE_DIR}
-	yarn config set cache-folder ${YARN_CACHE_DIR}
+	pnpm install --frozen-lockfile
 	if [ -z ${CIRCLE_BRANCH} ]; then \
-	  yarn install; \
+	  pnpm install; \
 	else \
-	  yarn install --frozen-lockfile; \
+	  pnpm install --frozen-lockfile; \
 	fi
-	# Install a symlink of the working directory as @codahq/packs-sdk, so that the sample code compiles.
-	yarn unlink || true # Remove any existing links providing @codahq/packs-sdk
-	yarn link  # Provide @codahq/packs-sdk from this directory
-	yarn link "@codahq/packs-sdk"  # Consume the link whenever @codahq/packs-sdk is imported
```

Comment:
Replaced this `yarn link` dance with a subpackage as well

Reply:
I think this can be simplified further.  IIUC, pnpm uses `--frozen-lockfile` by default on CI.  Also I notice you already have that on L27, so I think you can replace the entire if/else here.

Reply:
I think the extra call with `--frozen-lockfile` was making me think CI detection wasn't working. Simplified!